### DISCUSSION TITLE
How does Armeria integrates with java lightweight threads via spring boot tomcat

### DISCUSSION THREAD

**Discussion by khekrn** [line/armeria#5406] [General]
I have a spring boot application running with Java 21 and leveraging tomcat and lightweight threads, currently it’s written as REST service. Right now I’m exploring the GRPC service and found armeria,  curious to know if reactive stack armeria integrates with tomcat service which uses java lightweight threads via spring boot ?

**Comment by ikhoon** [comment]
#4911 We do not support virtual threads yet. Tomcat service is executed on the traditional blocking threads. However, virtual threads are one of our milestones for the first half of the year. We will try to support it soon. 💪

**Reply by khekrn** [reply]
Thanks @ikhoon !!
